ws/protomsg.go

```diff
@@ -28,14 +28,32 @@ const (
 // to which protocol the message should be routed.
 type ProtoHdr struct {
 	// Proto defines which protocol this message belongs
-	// to (required).
+	// to (required). E.g.:
+	// * MenderShell for remote terminal
+	// * MenderConfiguration for configuration of devices
+	// * MenderMonitoring for the stats and monitoring
```

Comment:
This also seems superfluous as the documentation for the different types should be placed by their definition. (Notice that this field is of `ProtoType`).